### PR TITLE
Add icons and improve README layout

### PR DESCRIPTION
**Pull request by mstuttgart**
Hi @jdinhify , thanks for this awesome theme! Good job! :grin: 

This PR add the follow changes:

* [NEW] Add a icon to theme (to show in VS Code marketplace) [here](https://github.com/mstuttgart/vscode-theme-gruvbox/blob/master/images/icon.png).
* [IMP] Add Icon, menu and badges in README. Improve section in REAME layout. Please, see new README [here](https://github.com/mstuttgart/vscode-theme-gruvbox/blob/master/README.md
).
* [NEW] Add me in 'Contributors' section. :grimacing: 

Thanks!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,7 +1,55 @@
...
+
+Launch *Quick Open*
+  - <img src="https://www.kernel.org/theme/images/logos/favicon.png" width=16 height=16/> <a href="https://code.visualstudio.com/shortcuts/keyboard-shortcuts-linux.pdf">Linux</a> `Ctrl+P`
+  - <img src="https://developer.apple.com/favicon.ico" width=16 height=16/> <a href="https://code.visualstudio.com/shortcuts/keyboard-shortcuts-macos.pdf">macOS</a> `⌘P`
+  - <img src="https://www.microsoft.com/favicon.ico" width=16 height=16/> <a href="https://code.visualstudio.com/shortcuts/keyboard-shortcuts-windows.pdf">Windows</a> `Ctrl+P`
+
+Paste the following command and press `Enter`:
+
+```
+ext install jdinhlife.gruvbox
```
Comment (jdinhify) [inline README.md]: This is awesome, would help lots of peeps install the correct thing

File: package.json [hunk 2]
```diff
@@ -18,6 +18,7 @@
 	"categories": [
 		"Themes"
 	],
+	"icon": "images/icon.png",
```
Comment (jdinhify) [inline package.json]: Please add yourself to the contributors list here as well 🙏

File: README.md [hunk 3]
```diff
@@ -1,7 +1,55 @@
...
+</p>
+
+<p align="center">
+  <a href="#about">About</a> |
+  <a href="#installation">Installation</a> |
+  <a href="#variants">Variants</a> |
+  <a href="#contributing">Contributing</a> |
+  <a href="#release-notes">Release Notes</a> | 
+  <a href="#credits">Credits</a>
+</p>
```
Comment (jdinhify) [inline README.md]: Is it possible to make these all markdown? Would make things a bit easier to read.

I'm alright with not having those center-aligned stuff as long as you're ok with it.
If it's not possible, it's alright though

File: package.json (hunk 2 above)
Comment (mstuttgart) [reply package.json]: Added!

File: README.md (hunk 3 above)
Comment (mstuttgart) [reply README.md]: @jdinhify hi! I use this template in my projects, but I can change it. 

I could not convert the icon in markdown (change position made the REAME ugly), but I convert menu in markdown. What do you think? See here: ttps://github.com/mstuttgart/vscode-theme-gruvbox/tree/add-icons

Comment (jdinhify) [reply README.md]: Thanks, let's use this. Not having menu is ok since the README is small anyway. I'll merge this PR and deploy soon when I have time.

**Pull request merged**